package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "simplerestclients",
-  "version": "0.2.11",
```

Comment:
Appreciate the help on this, but we'll bump the version ourself when we do a release. :D . Please don't include as part of the PR.

Reply:
Fixed